### DISCUSSION TITLE
Add ability to change playstyle for osu!catch on mobile

### DISCUSSION THREAD

**Discussion by ianpep** [ppy/osu#29315] [Ideas]
At the moment, osu!catch is locked to a single playstyle, of having separate buttons for both dashing left/right, and walking left/right. This is contrary to desktop, where usually one hand is controlling the catcher and the other controls the dashing. This makes switching from desktop to mobile very difficult. I suggest adding a dropdown to switch playstyles, similar to the selector for taiko, where you can switch to a playstyle closer to desktop, with an option to mirror it for those who play with the other hand
![IMG_8975](https://github.com/user-attachments/assets/550ba9b0-07d4-4c12-9b65-fd7507479355)

**Comment by peppy** [comment]
We do hope to allow customisation of this in the future!

**Comment by Hung696** [comment]
I want to try it out

**Comment by Hung696** [comment]
I want to try it out
